symfony/src/Entity/User.php

```diff
@@ -29,66 +37,102 @@ class User implements UserInterface
      * @ORM\GeneratedValue
      * @ORM\Column(type="integer")
      */
-    private $id;
+    private ?int $id;
```

Comment:
User ID kan niet null zijn

Reply:
Is standaard null tot dat flush wordt doorgevoerd. Met make:entity staat deze ook standaard op ?int in de getter

Reply:
Maker bundle is zeker niet de referentie 😄 

@ORM\GeneratedValue dus idd null tot flush maar altijd beschikbaar op profile. Bij validatie zou daarbij ID nooit meegenomen moeten worden.